scripts/toaster_configuration

```diff
@@ -0,0 +1,153 @@
+#!/bin/sh
+#
+# Copyright 2015-2016 Mentor Graphics Corporation
+#
+# This file is licensed under the terms of the GNU General Public License
+# version 2.  This program  is licensed "as is" without any warranty of any
+# kind, whether express or implied.
+
+echo "Creating toaster configuration"
+
+BUILDDIR=$*
+
+BBLAYERS=
+configured_layers () {
+   tac "$BUILDDIR/conf/bblayers.conf" | \
+   sed -n -e '/^"/,/^BBLAYERS = /{ /^BBLAYERS =/d; /^"/d; p;}' | \
+   awk {'print $1'}
+}
+
+configured_layers | while read layer; do
+   if [ -d "$layer" ]; then
+       #Once testing is done, remove this if.
+       if [ -d "$layer/.git" ]; then
+            rm -fr "$layer/.git"
+       fi
+       cd "$layer"
+       if [ ! -d "$PWD/.git" ]; then
+            git init . > /dev/null 2>&1
+            sleep 1
+            git add . > /dev/null 2>&1
+            sleep 1
+            git commit -m "Initial commit" > /dev/null 2>&1
+            sleep 1
+            cat <<EOF >> .git/config
+[remote "mel"]
+        url = git://"$layer".git
+        projectname = meta-fsl-arm.git
+EOF
+       fi
+       cd - > /dev/null 2>&1
+   fi
+done
+
+create_toaster_configuration() {
+DISTRO=$(grep "DISTRO =" "$BUILDDIR/conf/local.conf"  | awk {'print $3'} | sed -e s:\'::g)
+MACHINE=$(grep -i "MACHINE ??=" "$BUILDDIR/conf/local.conf" | awk {'print $3'} | sed -e s:\"::g)
+cat <<EOF > "$BUILDDIR"/toasterconf.json
+{
+    "config": {
+        "MACHINE"    : "$MACHINE",
+        "DISTRO"     : "$DISTRO",
+        "IMAGE_INSTALL_append": "",
+        "PACKAGE_CLASSES": "package_ipk",
+        "EXTERNAL_TOOLCHAIN": "",
+        "MGLS_LICENSE_FILE": "",
```

Comment:
@kergoth In the toaster configuration where I built console-image, This is what I had 
"EXTERNAL_TOOLCHAIN": "/home/sujith/codebench-linux-install-2014.11-96-arm/codebench",
"MGLS_LICENSE_FILE": "8224@svr-alm-eng-03.alm.mentorg.com",
I didn't added the LM_LICENSE_FILE. Intention of keeping this variable empty is so that user can point to the path in the toaster UI.